.changie.yaml

```diff
@@ -8,7 +8,9 @@ kindFormat: '### {{.Kind}}'
 changeFormat: '* {{.Body}}'
 kinds:
   - label: Breaking Changes
-    auto: major
```

Comment:
Since we're under `1.0` we can make breaking changes in minor versions so I changed it.